README.md

```diff
@@ -1,2 +1,28 @@
-# autopsy-packager
+# autopsy-packager ![Github Workflows](https://github.com/labcif/autopsy-packager/actions/workflows/packaging.yml/badge.svg?branch=main)
+
 Automatic Github Workflows packager for autopsy
+
+## Installation
+
+Before the installation process of `autopsy` package, make sure you have access to the following requirements:
```

Comment:
Be sure to have the following requirements before starting any installation process of `Autopsy`:

Reply:
Done